api/inc/svc_exports.h

```diff
@@ -116,6 +116,7 @@
 /* SVC immediate values for hardcoded table (call from unprivileged) */
 #define UVISOR_SVC_ID_UNVIC_OUT        UVISOR_SVC_FIXED_TABLE(0, 0)
 #define UVISOR_SVC_ID_REGISTER_GATEWAY UVISOR_SVC_FIXED_TABLE(3, 0)
+#define UVISOR_SVC_ID_BOX_INIT_NEXT    UVISOR_SVC_FIXED_TABLE(5, 0)
```

Comment:
Secure gateway handlers. See [here](https://github.com/Patater/uvisor/commit/3f4e9f03a1322048c1216538cddf40be1985f419#diff-af8f010563b390003c76aa7d9829538aR133). Yup a comment can do, just a line for every unused slot, something like:

```C
#define UVISOR_SVC_ID_UNVIC_OUT        UVISOR_SVC_FIXED_TABLE(0, 0)
/* Deprecated: UVISOR_SVC_ID_CX_IN     UVISOR_SVC_FIXED_TABLE(1, x) */
/* Deprecated: UVISOR_SVC_ID_CX_OUT    UVISOR_SVC_FIXED_TABLE(2, x) */
#define UVISOR_SVC_ID_REGISTER_GATEWAY UVISOR_SVC_FIXED_TABLE(3, 0)
```

and 

```C
".word  unvic_gateway_out\n"
/* Deprecated:  secure_gateway_in */
/* Deprecated:  secure_gateway_out */
".word  vmpu_register_gateway\n"
```